tutorials/beginners-cookbook/index.md

```diff
@@ -79,7 +79,7 @@ region.
 
 ![Raster](map-algebra.png)
 
-Image sources: https://gisgeography.com/spatial-data-types-vector-raster/
+[Image sources](https://gisgeography.com/spatial-data-types-vector-raster/)
```

Comment:
How about making is a little more explicit?:

\_Conceptual figures sourced from \[GISGeography\]\(https://gisgeography.com/spatial-data-types-vector-raster/)_

Reply:
Done